docs/server-side-configuration.md

```diff
@@ -8,25 +8,25 @@ The server-side part of the extension is the `DevExtreme.AspNet.Data.dll` assemb
 
 ## Custom Model Binder
 
-The server needs a custom model binder that will receive data loading options from the client and bind them to the model. The following links show how to implement it.
+The server needs a custom model binder that receives data loading options from the client and binds them to the model. Refer to the links below to learn how to implement it.
 
+- [ASP.NET Core](https://github.com/DevExpress/DevExtreme.AspNet.Data/blob/master/net/Sample/DataSourceLoadOptions.cs)
 - [Web API](https://github.com/DevExpress/devextreme-examples/blob/17_2/datagrid-webapi/datagrid-webapi/DataSourceLoadOptions.cs)
 - [ASP.NET MVC 5](https://github.com/DevExpress/devextreme-examples/blob/17_2/datagrid-mvc5/datagrid-mvc5/DataSourceLoadOptions.cs)
-- [ASP.NET Core MVC](https://github.com/DevExpress/DevExtreme.AspNet.Data/blob/master/net/Sample/DataSourceLoadOptions.cs)
 
-**NOTE:** If you use `DevExtreme.AspNet.Data` along with [DevExtreme ASP.NET MVC Controls](https://js.devexpress.com/Documentation/Guide/ASP.NET_MVC_Controls/), you do not need to implement the custom model binder, because it is already done in the ASP.NET MVC Controls library.
+**NOTE:** If you use `DevExtreme.AspNet.Data` along with [DevExtreme-based ASP.NET Core controls](https://docs.devexpress.com/AspNetCore/400263) or [DevExtreme ASP.NET MVC 5 controls](https://docs.devexpress.com/DevExtremeAspNetMvc/400943/), you do not need to implement the custom model binder, because these libraries already include required binders.
```

Comment:
```suggestion
**NOTE:** If you use `DevExtreme.AspNet.Data` with [DevExtreme-based ASP.NET Core controls](https://docs.devexpress.com/AspNetCore/400263) or [DevExtreme ASP.NET MVC 5 controls](https://docs.devexpress.com/DevExtremeAspNetMvc/400943/), you do not need to implement the custom model binder because it is included in these libraries.
```